Exiled.Events/EventArgs/Player/ChangingMoveStateEventArgs.cs

```diff
@@ -54,11 +57,19 @@ public ChangingMoveStateEventArgs(Player player, PlayerMovementState oldState, P
         /// <summary>
         ///     Gets or sets the new state.
         /// </summary>
-        public PlayerMovementState NewState { get; set; }
+        // TODO: remove setter
+        public PlayerMovementState NewState
+        {
+            get;
+            [Obsolete("Functional was removed due desync problems")]
```

Comment:
`Setters was removed due to desync problems.`

Reply:
fixed